js/runner.go

```diff
@@ -601,9 +603,8 @@ func (u *ActiveVU) RunOnce() error {
 
 func (u *VU) runFn(
 	ctx context.Context, isDefault bool, fn goja.Callable, args ...goja.Value,
-) (goja.Value, bool, time.Duration, error) {
+) (v goja.Value, isFullIteration bool, t time.Duration, err error) {
```

Comment:
I remember you being against named return values in functions shorter than this one :)

Is there a particular requirement for it here or you're OK with it now? I'm fine with it either way, just curious.

Reply:
I need to overwrite the `err`, there is practically no other way :) unless I make another smaller function just for that :)

Reply:
Also, I am mostly against naked returns ;) I am perfectly fine with the named returned variables